share-on-mastodon.js

```diff
@@ -1,46 +1,53 @@
 // Get the page title to use if a title isn't set.
```

Comment:
If you like, you can provide some general credits yourself in this file, since some people might just use this file instead of pulling the whole repository.

```suggestion
/**
 * @author Micah Ilbery
 * @version 1.2.0
 * @license GPL-3.0
 * @url https://github.com/micahilbery/share-on-mastodon
 */

// Get the page title to use if a title isn't set.
```